Cargo.toml

```diff
@@ -19,7 +19,7 @@ appveyor = { repository = "alexcrichton/tokio-core" }
 [dependencies]
 bytes = "0.4"
 log = "0.3"
-mio = "0.6.5"
+mio = { git = "https://github.com/asomers/mio", branch = "aio6" }
```

Comment:
I'll delete this part of the diff as soon as https://github.com/carllerche/mio/pull/586 merges